src/Moryx.Runtime.Endpoints/Common/HostHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) 2021, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+#if HAVE_SYSTEM_MANAGEMENT
+using System;
+using System.Linq;
+using System.Management;
+#endif
+
+namespace Moryx.Runtime.Endpoints.Common
+{
+    /// <summary>
+    /// Extensions to retrieve system information
+    /// </summary>
+    internal static class HostHelper
+    {
+        /// <summary>
+        /// Returns the amount of physical memory installed
+        /// </summary>
+        /// <returns></returns>
+        public static ulong PhysicalMemory()
+        {
+#if HAVE_SYSTEM_MANAGEMENT
```

Comment:
This compiler flag does not make sense anymore. It does not exist in the project and the types are not availble in >=net5